db/seeds.rb

```diff
@@ -19,22 +19,23 @@
+I work full time, but run Humble Hearts as an unpaid volunteer on nights and weekends, feeding the homeless 3-4 nights a week, collecting food, clothing, furniture and other items, making deliveries and creating special events to support them. It is my goal to one day run Humble Hearts full-time and be able to quit my day job, so I can dedicate all my efforts to helping people who are struggling everyday to find a reason to survive.
 
-TeamMember.create(name: "Ariella Boyd" , title: "Secretary", photo_url: "", bio: "I am a recent graduate of DePaul University’s College of Science and Health, majoring in Psychology with a concentration in Human Services, and a minor in Anthropology. I currently attend The Chicago School of Professional Psychology where I am working towards obtaining a Master’s Degree in Forensic Psychology. I have worked and volunteered in both legal and counseling settings. While I am not yet able to offer therapy to clients of Humble Hearts, in the meantime I am able to offer the professional skills and strategies that I have gained throughout my experience working with emotionally distressed individuals and my competence in resources for those in need of mental health assistance, especially considering the Medicaid population which is highly underserved. I am especially interested in aiding those that we service in getting their educations back on track, as many of the opportunities in today’s job market require at least a high school diploma or GED. Through providing the resources and connections necessary for both the homeless youth and adult populations to obtain such certifications, we can better prepare them for opportunities that may arise or that they may gain the education and confidence to seek. Looking to help those in need from a holistic approach will allow me to contribute meaningful changes and positive growth for those we help.")
+I will never forget where I came from and that is why I named my company “Humble Hearts”.  It is a daily, personal reminder to me to stay focused and genuinely help those in need. My goal is to invest in those that I help individually so that they get all the attention needed to feel special and know that someone relates to their situation. I want to change humiliation into dignity with comforting smiles. To show that no matter where you came from, it doesn’t mean you have to remain there; it just means you have to work to get to that new beginning a little harder than others. Humble Hearts is a part of that beginning, and I am willing to help because I believe in chances, and I believe that God wants us all to help each other with compassion. Humble Hearts is not a religious foundation, but because I carry God in me, this leads me to help mankind – We are a humanitarian charity work in God’s name" )
```

Comment:
@wiliajc87 The end of line 29 needs some work (currently: Humble Hearts is not a religious foundation but because I carry God in me this leads me to help mankind – We are a humanitarian charity work in God’s name)